lib/collection/src/collection/mod.rs

```diff
@@ -531,16 +531,33 @@ impl Collection {
                     sync: true,
                     method: None,
                 };
+
                 if check_transfer_conflicts_strict(&transfer, transfers.iter()).is_some() {
                     continue; // this transfer won't work
                 }
+
+                // TODO: Should we, maybe, throttle/backoff this requests a bit?
+                if let Err(err) = replica_set.health_check(replica_id).await {
```

Comment:
Do we eventually want to improve this call with something that actually checks on the remote that the shard is ready and active?

Or do we solely want to depend on consensus to synchronize this state, and leave this call as simple as possible?

I assume the latter, right?